src/shim/actors/mod.rs

```diff
@@ -8,3 +8,8 @@ pub mod multisig;
 pub mod verifreg;
 
 pub use common::*;
+
+pub mod state_load;
+pub use state_load::*;
+mod version;
+pub use version::*;
```

Comment:
How about adding `prelude` and `use  crate::shim::actors::prelude::*` in other places 
```rust
pub mod prelude {
    pub use super::state_load::*;
}
```

Reply:
I'm not convinced that adding a prelude with a single wildcard import is useful. I already used `use crate::shim::actors::state_load::*` in places where lots of different actor state loading was required.

We might want to add a bigger prelude later, depending on the usage, with changing the visibility of other modules here, but I wouldn't do it now.